metric_providers/psu/energy/ac/xgboost/machine/provider.py

```diff
@@ -9,18 +9,25 @@
 
 #pylint: disable=import-error, wrong-import-position
 import model.xgb as mlmodel
-from global_config import GlobalConfig
 from metric_providers.base import BaseMetricProvider
 
 class PsuEnergyAcXgboostMachineProvider(BaseMetricProvider):
-    def __init__(self, resolution, **_):
+    def __init__(self, resolution, HW_CPUFreq, CPUChips, CPUThreads, TDP,
```

Comment:
is there the possibility to add these variables not by position, but by keyword? If we every by accident change the order in the config.yml this will bite us in the ass